src/java.base/solaris/native/libjvm_db/libjvm_db.c

```diff
@@ -1327,7 +1332,7 @@ int Jget_vframe(jvm_agent_t* J, int vframe_no,
   int32_t   err;
 
   if (vframe_no >= N->vf_cnt) {
-     (void) sprintf(name, "Wrong inlinedMethod%1d()", vframe_no);
+     (void) snprintf(name, size, "Wrong inlinedMethod%1d()", vframe_no);
```

Comment:
While you're here, please correct the size argument on line 1340:
```c
  err = name_for_methodPtr(J, vf->method, name + 1, size - 1);
```
(see lines 1097 and 1100 for comparison).

Reply:
Updated.